### PR TITLE
doc: Let's say it's KonnectorLibs that handle errors

### PR DESCRIPTION
**Pull request by Crash--**
Konnectors errors should be listed in Konnector-Libs. 
Also, added a link to Harvest since it's Harvest that 
handle errors now.

### CODE REVIEW

File: src/tutorials/konnector/going-further.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ A basic connector workflow involves:
 
 #### Error handling
 
-If your connector hits an issue fetching or saving the data, it can return an error code by throwing it as an error. The error codes are defined inside the [Cozy Home][] application and will display an explicit error to the user:
+If your connector hits an issue fetching or saving the data, it can return an error code by throwing it as an error. The error codes are defined inside [cozy-konnector-libs][] and [Cozy Harvest Lib][https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/locales/en.json#L124-L208] will display an explicit error to the user:
```
Comment (sblaisot) [inline src/tutorials/konnector/going-further.md:39]:
```patch
- [cozy-konnector-libs][]
+ [cozy-konnector-libs][https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/src/helpers/errors.js]
```

**Review by sblaisot — APPROVED** [review]
One missing link to add, otherwise LGTM

File: src/tutorials/konnector/going-further.md (hunk 1 above)
Comment (Crash--) [reply src/tutorials/konnector/going-further.md:39]: already defined in the end of the file ;). This is not needed.

Comment (sblaisot) [reply src/tutorials/konnector/going-further.md:39]: Oh right, I missed that, sorry.
I just noticed the other link is broken

```patch
- [][]
+ []()

Comment (sblaisot) [reply src/tutorials/konnector/going-further.md:39]: And probably using an [anchor link](https://gist.github.com/rachelhyman/b1f109155c9dafffe618) would be better

**Pull request merged**